pkg/skaffold/kubernetes/debugging/transform_nodejs_test.go

```diff
@@ -72,6 +73,20 @@ func TestNodeTransformer_Apply(t *testing.T) {
 			},
 			debugConfig: types.ContainerDebugConfiguration{Runtime: "nodejs", Ports: map[string]uint32{"devtools": 9229}},
 		},
+		{
+			description:   "entrypoint with PATH and brk",
```

Comment:
nit: not sure what brk means I would say something like "entrypoint with PATH and wait enabled in debugger metadata"

Reply:
the inspiration came from [here](https://github.com/pkoutsovasilis/skaffold/blob/a1bd2365ce9a6f40d92e44dea3935816ea8001fa/pkg/skaffold/debug/transform_nodejs.go#L52), but sure I can change it 🙂